freeze/src/main.rs

```diff
@@ -0,0 +1,24 @@
+use std::collections::HashMap;
+
+use rustpython_vm as vm;
+
+fn main() -> vm::pyobject::PyResult<()> {
+    let vm = vm::VirtualMachine::new(vm::PySettings::default());
+
```

Comment:
Does this need to be its own crate? Can it be a file in `examples/`?

Reply:
The important part in the example is `default-features = false`. Not sure how to do that in the `examples/`...

Reply:
What about moving the whole directory into `examples`? It feels weird to have an example crate as part of the workspace...